docs/Superoperator representations.md

```diff
@@ -12,49 +16,565 @@ where $i$ is a row and $j$ is a column index.
+
+The importance of understanding reshuffling can be understood as understanding the relationship between 
+$${\rm vec}(G)\otimes {\rm vec}(\Gamma) \quad {\rm and} \quad  {\rm vec}(G\otimes\Gamma)$$
+where $G$ and $\Gamma$ are matrices that act on $\mathbb H_A$ and $\mathbb H_B$ respectively, as explaind in [VECQO].
+
+### A note on numerical implementations
+Most linear algebra (or tensor) libraries have the ablity to `reshape` a matrix and `swapaxes` (or sometimes it is called `permute_dims`). 
+
+If you are trying to reshuffle indicies usually the first job is to write your matrix in tensor form. This requires reshaping a $d_A^2\times d_B^2$ matrix into a $d_A\times d_A\times d_B \times d_B$ tensor. Next you `permute1_dims` or `swapaxes`. Often $d_A = d_B$ so we `reshape` to a Matrix that has the same dimensions as the orignal  $d_A^2\times d_A^2$ matix.
+
+
+## The $n$-qubit Pauli basis 
+ The $n$-qubit Pauli basis is denoted $\mathcal P^{\otimes n} $ where $\mathcal  P = \{ I, X, Y, Z \}$ are the usual Pauli matricies. It is an operator baisis for the $d = 2^n$ dimensional Hilbert space and there are $d^2 = 4^n$ operators in $\mathcal P^{\otimes n} $. If one divides all the operators by $\sqrt{d}$ the basis is orthonormal with respect to the Hilbert-Schmidt inner product.
+ 
+It is often convenient to index the $d^2$ operators with a single label, e.g. $P_1=I^{\otimes n},\, \ldots,\, P_{d^2}= Z^{\otimes n}$  (or $P_0=I^{\otimes n}$ if you like zero indexing). In anycase, as these operators are Hermitian and unitary they obey $P_i^2=I^{\otimes n}$.
+ 
+To be explicit, for two qubits $d=4$ and we have 16 operators e.g. $\{II, IX, IY, IZ, XI, XX, XY, ..., ZZ\}$ were $II$ should be intepreted as $I\otimes I$ etc. The single index would be $\{P_1, P_2, P_3, P_4, P_5, P_6, P_7, ..., P_{16}\}$.
+
+## Quantum channels in the Kraus decomposition (or operator-sum representation)
+A completely positive map on the state $\rho$ can be written using a set of Kraus operators $\{ M_k \}$ as
+
+
+$$\rho' =\mathcal E (\rho) = \sum_{k=1}^N M_k \rho M_k^\dagger, $$
+where $\rho'$ is the state at the output of the channel.
+
+If $\sum_k M_k^\dagger M_k= I $ the map is trace preserving. It turns out that $N\le d^2$ where $d$ is the Hilbert space dimension e.g. $d=2^n$ for $n$ qubits.
+
+
+## Kraus to process matrix (aka $\chi$ or chi matrix)
+We choose to represent the process matrix in the Pauli basis. So we expand each of the Kraus operators in the $n$ qubit Pauli basis 
+
+$M_k = \sum^{d^2}_{j=1}c_{kj}\,P_j$ 
+
+where $\mathcal P_j \in \mathcal P ^{\otimes n}$.
+
+Now the channel $\mathcal E$ can be written as
+
+$\mathcal E (\rho) = \sum_{i,j=1}^{d^2} \chi_{i,j} P_i\rho P_j ,$
+
+where 
+
+$$\chi_{i,j} = \sum_k c_{k,i} c_{k,j}^*$$ 
+
+is an element of the process matrix $\chi$  of size $d^2 \times d^2$. The if the channel is CP the process matrix is a Hermitian and positive semidefinite. 
+
+
+## Kraus to Pauli Transfer matrix
```

Comment:
I am not a big fan of the name "Pauli Transfer Matrix" -- it is just the Liouville representation using the Pauli basis.  I am happy to compromise and say leave the PTM name in, but make a comment it is just the Liouville representation with a Pauli basis, much like the chi matrix is the Choi Matrix in the Pauli basis.

Reply:
Im happy to change it. 

In the Superoperator section I'll mention the basis.

Reply:
FWIW, the term "Pauli transfer matrix" seems to have already caught on. I think it is worth mentioning that it is the Liouville representation in the Pauli basis, but I wouldn't otherwise fight this trend.